docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -449,6 +447,9 @@ class DebugPage extends React.PureComponent<Props, State> {
   };
 
   private validateEndpointPath(endpointPath: string) {
+    if (!endpointPath) {
+      throw new Error(`You should set the ENDPOINT PATH.`);
```

Comment:
nit: How about just using single(`'`) quotes here? :-)

Reply:
Oops~ fixed. Thank you. 😉